lib/Parser/Parse.h

```diff
@@ -780,7 +780,7 @@ class Parser
     template<bool buildAST> ParseNodePtr ParseMemberList(LPCOLESTR pNameHint, uint32 *pHintLength, tokens declarationType = tkNone);
     template<bool buildAST> IdentPtr ParseSuper(bool fAllowCall);
 
-    bool IsTerminateToken();
+    bool IsTerminateToken(bool inTerminate);
```

Comment:
Nit : would it better to understand if this is called as tkInAsTerminate?

Reply:
Can't we just reuse the name fAllowIn?

Reply:
I've renamed it to fAllowIn